lib/condorSecurity.py

```diff
@@ -242,7 +349,27 @@ def get(self, context, feature):
 
 
 class GSIRequest(ProtoRequest):
```

Comment:
Should the inherited class be mentioned in the docstring?

Reply:
If you mean the parent, it's in the signature